internal/namespaces/mnq/v1beta1/custom_nats.go

```diff
@@ -0,0 +1,98 @@
+package mnq
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"reflect"
+
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	mnq "github.com/scaleway/scaleway-sdk-go/api/mnq/v1beta1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+type natsContext struct {
+	Description string `json:"description"`
+	URL         string `json:"url"`
+
+	// CredentialsPath is a path to file containing credentials
+	CredentialsPath string `json:"creds"`
+}
+
+func natsContextFrom(account *mnq.NatsAccount, credsPath string) []byte {
+	ctx := &natsContext{
+		Description:     "Nats context created by Scaleway CLI",
+		URL:             account.Endpoint,
+		CredentialsPath: credsPath,
+	}
+	b, _ := json.Marshal(ctx)
+	return b
+}
+
+type CreateContextRequest struct {
+	NatsAccountID   string
+	ContextName     string
+	CredentialsName string
+	Region          scw.Region
+}
+
+func aliasCreateContextCommand() *core.Command {
```

Comment:
Change the function name, this one is remaining from when I copied alias command